src/randoop/main/GenTests.java

```diff
@@ -429,264 +399,335 @@ public boolean handle(String[] args) throws RandoopTextuiException {
         }
       }
     }
+    
+    ExecutionVisitor visitor;
+    if (visitors.isEmpty()) {
```

Comment:
I guess I'm responsible for forcing the visitor to be non-null, but otherwise the single visitor within the test generator is as it was.

I'd be OK removing ExecutionVisitors altogether. Whether there is a good use case for them is not clear.